driver/controller.go

```diff
@@ -955,7 +955,11 @@ func extractStorage(capRange *csi.CapacityRange) (int64, error) {
 	}
 
 	if requiredSet && !limitSet && requiredBytes < minimumVolumeSizeInBytes {
-		return 0, fmt.Errorf("required (%v) can not be less than minimum supported volume size (%v)", formatBytes(requiredBytes), formatBytes(minimumVolumeSizeInBytes))
+		d.log.WithFields(logrus.Fields{
+			"requiredBytes":            formatBytes(requiredBytes),
+			"minimumVolumeSizeInBytes": formatBytes(minimumVolumeSizeInBytes),
+		}).Warn("requiredBytes is less than minimum supported volume size, setting requiredBytes default to minimumVolumeSizeBytes")
+		return minimumVolumeSizeInBytes, nil
```

Comment:
Nit: Let's say "minimum volume size" to not give the impression that `minimumVolumeSizeBytes` is a known value from the CSI spec (which `requiredBytes` is only).